src/Query/QueryManager.ts

```diff
@@ -0,0 +1,78 @@
+import { IQueryBuilder } from "./QueryBuilder";
+import { IQueryResponse } from "./IQueryResponse";
+import { IModel, IModelData } from "..";
```

Comment:
Update import to use `../Model` instead of just `..` to avoid confusion, and avoid an occasional issue with cyclical imports caused by webpack barreling.

Reply:
fixed